rJavaPackageExample.Rproj

```diff
@@ -0,0 +1,21 @@
+Version: 1.0
```

Comment:
This file is generated when you run `devtools::create("path/to/package/pkgname")` command to create a new R package template (following the advice in ["R packages" book](http://r-pkgs.had.co.nz/package.html#getting-started)). This is a configuration file for RStudio but since it is a de facto standard R editor now, I decided to leave it.